sqldelight-sample/android/build.gradle

```diff
@@ -46,8 +43,24 @@ android {
 dependencies {
   implementation deps.kotlin.stdlib.jdk
   // TODO implementation deps.androidx.sqliteFramework
-  implementation 'android.arch.persistence:db-framework:1.1.0'
+  implementation 'androidx.sqlite:sqlite:2.0.0-rc01'
 
   implementation "com.android.support:recyclerview-v7:27.1.1"
-  implementation "com.squareup.sqldelight:android-driver:${versions.sqldelight}"
+  implementation ("com.squareup.sqldelight:android-driver:${versions.sqldelight}") {
+    exclude group: 'com.squareup.sqldelight', module: 'runtime-jdk'
```

Comment:
this is because of the two different runtime jdk artifacts. Definitely gotta figure this out before 1.0